logging/cluster-logging-exported-fields.adoc

```diff
@@ -13,3 +13,4 @@ To search these fields from Elasticsearch and Kibana, use the full dotted field
 
```

Comment:
Can't comment on line 8:
The following fields can be present in log records exported by OpenShift Logging system.
An article missing?
The following fields can be present in log records exported by THE OpenShift Logging system.

Reply:
Updated to "exported by OpenShift Logging." Thanks.